### ISSUE TITLE
Bad web3 dependency definition on setup.py

### ISSUE THREAD

**Issue by lullis** [gnosis/gnosis-py#7]
It seems that gnosis as a whole still depends on web3 v4, but if one install the package through pip we get web3 v5. This means that, whenever I try to run importing anything from gnosis.eth, I get:

```
from web3.utils.threads import Timeout
ModuleNotFoundError: No module named 'web3.utils'
```

**Comment by Uxio0** [comment]
You are right @lullis, it has ben fixed in version `v1.6.2`

Thanks for your report! We will be migrating soon to web3 v5

**Comment by Uxio0** [comment]
dc59e7513b1272c04b843ee4e0295f79407408e6
